src/comment.ts

```diff
@@ -2,25 +2,31 @@ function headerComment(header) {
   return `<!-- Sticky Pull Request Comment${header} -->`;
 }
 
+function removeHeaderComment(body, header) {
+  return body.replace(`\n${header}`, '');
+}
+
 export async function findPreviousComment(octokit, repo, issue_number, header) {
   const { data: comments } = await octokit.issues.listComments({
     ...repo,
     issue_number
   });
   const h = headerComment(header);
-  return comments.find(comment => comment.body.startsWith(h));
+  return comments.find(comment => comment.body.includes(h));
 }
 export async function updateComment(octokit, repo, comment_id, body, header, previousBody?) {
+  const headerIdentifier = headerComment(header);
+  const updatedBody = previousBody ? `${removeHeaderComment(previousBody, headerIdentifier)}\n${body}` : body;
```

Comment:
```suggestion
  const updatedBody = previousBody ? `${previousBody}\n${body}` : `${headerComment(header)}\n${body}`;
```
will be enough.
please remove `removeHeaderComment` also.